clang/test/CodeGen/AArch64/inline-asm-ios.c

```diff
@@ -0,0 +1,23 @@
+// REQUIRES: aarch64-registered-target
```

Comment:
Is this a new folder? There should be a `lit.local.cfg` in there that takes care of this:

```
if "AArch64" not in config.root.targets:
    config.unsupported = True
```

If not, please add one.

Reply:
Yep, seems like I accidentally added a new folder. Targets other than AArch64 and Arm use a subfolder but still rely on `REQUIRES`. AArch64 just dumps everything into the top-level test folder and thus have to rely on `REQUIRES`. 

I now move the test up to `test/CodeGen` and stick with `REQUIRES`. Should be reworked but not in this patch.